app/routes/image-set.js

```diff
@@ -0,0 +1,39 @@
+import Ember from 'ember';
+import {imageTypes, genders} from 'lion-guardians/utils/units';
+
+var DEBUG = false;
```

Comment:
this would probably be nicer as a setting in config/environment.js. Much easier to a) globally apply, b) avoid accidentally committing it as "true", and c) guard against the value ever being true in production.